Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Pistols/mk80_pistol.yml

```diff
@@ -137,8 +299,84 @@
   - type: RMCProjectileAccuracy
     accuracy: 105
 
+- type: entity
+  parent: RMCBulletPistol9mmSquashHead
+  id: RMCBulletPistol9mmSquashHeadIncendiary
+  name: bullet (9mm Squash-Head, Incendiary)
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Heat: 35
+  - type: IgniteOnProjectileHit
+
+- type: entity
+  parent: RMCBulletPistol9mmSquashHead
+  id: RMCBulletPistol9mmSquashHeadWallPiercing
+  name: bullet (9mm Squash-Head, Wall-Piercing)
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    deleteOnCollide: false
+    damage:
+      types:
+        Piercing: 45
+  - type: RMCPenetratingProjectile
+
+- type: entity
+  parent: RMCBulletPistol9mmSquashHead
+  id: RMCBulletPistol9mmSquashHeadRubber
+  name: bullet (9mm Squash-Head, Rubber)
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Blunt: 2
+  - type: RMCStaminaDamageOnCollide
+    damage: 40
+
+- type: entity
+  parent: RMCBulletPistol9mmSquashHead
+  id: RMCBulletPistol9mmSquashHeadToxic
+  name: bullet (9mm Squash-Head, Toxic)
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Heat: 10
+  - type: AcidProjectile
```

Comment:
This is supposed to do 3 times damage to objects and walls, not 2.625
It's also missing the acid buildup, so it would probably be easier if you leave this out for now unless you want to code that